components/library/libxxhash/xxhsum.p5m

```diff
@@ -0,0 +1,37 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2024 Till Wegmüller
+#
+
+set name=pkg.fmri value=pkg:/sysutils/xxhsum@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.human-version value=$(HUMAN_VERSION)
```

Comment:
sysutils?
COMPONENT_FMRI  should be used here!

Reply:
no I am seperating the binary and library package as the library is used in many applications and the binary is statically compiled thus does not need the library.

Reply:
Please note that you can use `$(COMPONENT_FMRI)` here and set `COMPONENT_FMRI.xxhsum = sysutils/xxhsum` in the `Makefile`.

Reply:
Oh, thats a magic that i never used before. Switched.

Reply:
Not surprising because it is pretty new and not widely used yet.